### ISSUE TITLE
[PAGE ISSUE]: "exercise" spelt incorrectly as "exericise" in image name

### ISSUE THREAD

**Issue by parlough** [dart-lang/site-www#3614]
### Page URL

https://dart.dev/resources/dartpad-best-practices.html

### Page source

https://github.com/dart-lang/site-www/tree/master/src/resources/dartpad-best-practices.md

### Describe the problem

The alt text and image file name within the **II. Exercise** section has misspelt "exercise" as "exericise".

### Expected fix

The image [m_exericise.png](https://github.com/dart-lang/site-www/blob/master/src/_assets/image/dartpad-best-practices/m_exericise.png) should be renamed to `m_exercise.png` and the [reference to that image and the corresponding alt text](https://github.com/dart-lang/site-www/blob/master/src/resources/dartpad-best-practices.md?plain=1#L209) should be updated with the correct spelling.

**Comment by PurpleDDive** [comment]
Hello there!
I want to help with this issue, hope this issue hasn't been fixed already like the issue 3616.
Can I still be assigned?

**Comment by san-2015** [comment]
May I try to fix this issue?
